modules/creating-a-machine-pool-cli.adoc

```diff
@@ -10,8 +10,8 @@ You can create additional machine pools for your {product-title} (ROSA) cluster
 
 .Prerequisites
 
-* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
-* You logged in to your Red Hat account by using the `rosa` CLI.
+* You installed and configured the latest {product-title} (ROSA) CLI, `rosa` on your workstation.
```

Comment:
`rosa` is a parenthetical phrase here so it needs a closing comma as well: "...the latest {product-title} (ROSA) CLI, `rosa`, on your..."